src/Community/Block/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { Action, Content, Icon, Header, Wrapper } from './styles'
```

Comment:
I wouldn't call the styled container file `styles.js` though, sounds like we're talking about plain CSS or maybe Sass, etc. (especially when importing `from './styles'`.

How about `styled.js`? Or maybe even `styled_comps.js`. Small change, I know, but meaningful:

```suggestion
import { Action, Content, Icon, Header, Wrapper } from './styled'
```

Reply:
> Or maybe even `styled_comps.js`

Or `StyledComponents.js`, actually.

Reply:
or even `comps/styled.js` :p

Reply:
I'm ok either way, as long as they are all named the same and are in the same folder os the main component.

Reply:
OK, I vote for `comps/styled.js`. Seems like the shortest, cleanest option to me.

Reply:
One more consideration: We can replace styled components with something else in the future, so maybe we shouldn't make styles file name based on them.

Reply:
BTW I take that back. I just realized these files are already inside component named directories so I'd just rename them to `styled.js` (no further `comps/` dir).
About replacing this later, that's OK, we can rename the files again if needed. Who knows how the replacement will look, maybe these files will be totally gone anyway.